tools/hatch-cli/src/util.ts

```diff
@@ -471,6 +543,22 @@ export const createFromTemplate = async (
           fs.writeFileSync(rushConfigPath, rushConfigRawUpdated);
           await updateDockerComposition(templateName, monorepoRootDir, toShortName(name));
         }
+        if (clientSDKOptions != null) {
+          const clientSDKPackagePath = path.resolve(tempFilePath, 'package.json');
```

Comment:
might be good to refactor this out into its own method, this method is stating to get pretty long

Reply:
updated